src/server/routes/search.js

```diff
@@ -134,11 +134,6 @@ router.get('/reindex', auth.isAuthenticated, (req, res) => {
 		// TODO: This is hacky, and we should replace it once we switch to SOLR.
 		const trustedUsers = ['Leftmost Cat', 'LordSputnik', 'Monkey', 'iliekcomputers'];
 
-		const NO_MATCH = -1;
-		if (trustedUsers.indexOf(req.user.name) === NO_MATCH) {
-			throw new error.PermissionDeniedError(null, req);
-		}
-
```

Comment:
Did you mean to commit this?

Reply:
Oops. I'll undo this. I did this because I was changing users and refreshing index while testing. and it was taking time to add user in trustedUsers each time :P.

Reply:
Yep, totally understand. It's a far from ideal situation at the moment.
I just assumed you committed it by mistake :)

Reply:
This change hasn't been removed.